discovery/staticjson/lib.go

```diff
@@ -18,11 +18,10 @@
 // OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 // THE SOFTWARE.
 
-package swim
+package staticjson
```

Comment:
not really sure if this package name captures what it does.

1. It is not necessarily static as it reads the file every time you call `Hosts`
2. This name does not tell that it is reading the json from a file

I would suggest `jsonfile` as the package name.